hr_holidays_public/models/hr_holidays_public.py

```diff
@@ -135,26 +140,30 @@ class HrHolidaysPublicLine(models.Model):
         'Name',
         required=True,
     )
+
```

Comment:
Please don't add useless empty lines. It's discouraged by PEP8 and doesn't increase readability. Indentation does that.

Reply:
Noted!